2 nové infografiky (Emisie sveta na osobu a na HDP)

mukrop opened the pull request:
Nahlad najdete na nasledovnych adresach:
* https://sk-fakty-o-klime--preview-emisie-svet-na-osobu-hdp-jv3y6nt9.web.app/infografiky/emisie-svet-na-hdp
* https://sk-fakty-o-klime--preview-emisie-svet-na-osobu-hdp-jv3y6nt9.web.app/infografiky/emisie-svet-na-osobu

Prosim @kikinna o potvrdenie, ze infografiky su OK (a ze su spravne zaradene v grafickom repe).
Prosim @vrq-github alebo @rastislavsk o potvrdenie, ze to online vyzera tak, ako ma (ze text je ta verzia, ktora ma byt a podobne).
Potvrdenie idealne sposobom, ze na tento pull request udelite "approve" (na psolednej karte "code" je to velke zelene tlacitko "Submit review").

rastislavsk committed:
Emisie svetovych regionov prepocitane na osobu

preklad

rastislavsk committed:
Emisie svetovych regionov prepocitane na HDP

preklad

mukrop committed:
Pridaná infografika, opravy v texte

* Pridaná slovenská infografika.
* Opravené odkazy do slovníka.
* Opravené/zakomentované odkazy na ďalšie infografiky.
* Upravená kapitalizácia nadpisov.
* Upravený dátum publikovania.

mukrop committed:
Pridaná infografika, opravy v texte

* Pridaná slovenská infografika.
* Opravené odkazy do slovníka.
* Opravené/zakomentované odkazy na ďalšie infografiky.
* Upravená kapitalizácia nadpisov.
* Upravený dátum publikovania.

kikinna commented:
Grafiky jsou obecně v poho. Věci, které bych vyčetla jsou ty, co jsem dělala už já v českých :D 
Jen dvě připomínky. 

V emise-svet-na-hdp bych trochu odsunula název "Jižní a severní amerika" maličká kousíček doleva, aby se udělala trochu větší mezera mezi tímto popiskem a popiskem "Afrika" 
![image](https://user-images.githubusercontent.com/11555522/137317622-7ec177a9-3551-4dae-bd1c-c737ed53567e.png)


V grafice emisie-svet-na-osobu (a teoreticky i ...-na-hdp), by stálo za to "jak číst tento graf" vyrovnat odstavec. Ve verzi "-na-osobu" je to zarovnané do bloku až na poslední dva (!) řádky -- když už, tak se dělá jen jeden. Klidně zkus @vrq-github klidně trošku roztáhnout textový blok, ať se ti tam vejde i to číslo na posledním řádku.
Obdobně v grafice "-na-hdp" má tato sekce na prvních dvou řádky zbytečně velké mezery mezi slovy, myslím. Stojí za to zkusit trošku rozšířit / zúžit ten blok textu a možná se to spraví. Nebo zkrátit "bilionov" na "bil."
![image](https://user-images.githubusercontent.com/11555522/137318310-4f9a770e-598b-4552-8e37-758b19b3eef8.png)

A nakonec přeci jen tě poprosím, upravíš prosím barvu písma v tom bloku "jak číst tento graf" tak, aby byla třeba o 5-10 procentních bodů tmavší? (Např pokud je to šedá z 60 %, tak hodit třeba na 65-70%.) Ten kontrast co tam máme je mizernej. ))

kikinna commented:
@vrq-github Ještě si prosím zkontroluj, jestli datum verze u grafik je správné. Je tam tuším február, ale pokud na tom teď ještě pracuješ, tak bych tam dala nejpozdnější datum. Unless máte s Martinem jinou dohodu

mukrop commented:
@kikinna, nemame. Na datum verzie v infografike si autorita ty. Suhlasim, ze by tam mal byt datum poslednej Vierkynej upravy (v obidvoch).

LenkaSedlackova committed:
Mjanmarsko, pomlčka, percentá

@mukrop , ak posunieš Mitigačné opatrenia MMF, tu treba opraviť odkaz na túto IG v Komentáre ku grafu, predposledný bod

LenkaSedlackova committed:
medzery pred %

mukrop commented:
* [x] Odkomentovat link na Mitigacne opatrenia MMF.

vrq-github commented:
Skúšala som ten odstavec spraviť ako to máte vy, ale nejde mi to zarovnať po oboch stranách, aby tam neboli veľké medzery medzi písmenami
![ost](https://user-images.githubusercontent.com/77229161/141107613-a7198fc4-14e4-4875-9f61-5d237a8d4d24.png)

vrq-github commented:
Toto je najlepšie asi ako to viem @kikinna https://drive.google.com/drive/folders/1xHI_MPHbLRqLNTNgfeBQgNJI2RIKKhz5 a https://drive.google.com/file/d/1oDdDNZeitaLTvkFoL5gCDSal4Dmb0-3t/view?usp=sharing Dala som to zatiaľ do PDF na nahliadnutie, lebo to svg robi zlobu, aj ked to dobre uložím

mukrop committed:
Aktualizované grafiky, odkomentované odkazy

mukrop reviewed (commented):
Infografiky som aktualizoval podla tych, ktore linkovala Viera a upravil posledne drobnosti. Komentare od Kristi som presiel a skontroloval, ze su zapracovane.
Dakujem vsetkym zucastnenym za spolupracu :-).
